index.html

```diff
@@ -0,0 +1,154 @@
+          </div>
+        </div>
+      </section>
+      <section class="feynman">
+        <h2 class="feynman__title">The Feynman Technique</h2>
+        <p class="feynman__subtitle">Learn without forgetting</p>
+        <a  href="https://fs.blog/feynman-learning-technique/" class="link feynman__link" target="_blank">More &rarr;</a>
+      </section>
+      <section class="kaufman">
+        <h2 class="section__title">Ten principles of rapid skill acquisition</h2>
+        <p class="section__subtitle">by Josh Kaufman</p>
+        <ul class="table">
+          <li class="table__cell">
+            <h5 class="table__heading">1</h5>
+            <p class="table-text">Choose a lovable project</p>
+          </li>
+          <li class="table__cell">
+            <h5 class="table__heading">2</h5>
+            <p class="table-text">Focus your energy on one skill at a time</p>
+          </li>
+          <li class="table__cell">
+            <h5 class="table__heading">3</h5>
+            <p class="table-text">Define your target performance level</p>
+          </li>
+          <li class="table__cell">
+            <h5 class="table__heading">4</h5>
+            <p class="table-text">Deconstruct the skill into sub-skills</p>
+          </li>
+          <li class="table__cell">
+            <h5 class="table__heading">5</h5>
+            <p class="table-text">Obtain critical tools</p>
+          </li>
+          <li class="table__cell">
+            <h5 class="table__heading">6</h5>
+            <p class="table-text">Eliminate barriers to practice</p>
+          </li>
+          <li class="table__cell">
+            <h5 class="table__heading">7</h5>
+            <p class="table-text">Make dedicated time for practice</p>
+          </li>
+          <li class="table__cell">
+            <h5 class="table__heading">8</h5>
+            <p class="table-text">Create fast feedback loops</p>
+          </li>
+          <li class="table__cell">
+            <h5 class="table__heading">9</h5>
+            <p class="table-text">Practice by the clock in short bursts</p>
+          </li>
+          <li class="table__cell">
+            <h5 class="table__heading">10</h5>
+            <p class="table-text">Emphasize quantity and speed</p>
+          </li>
+        </ul>
+        <img class="kaufman__triangle rotation" src="images/kaufman-triangle.svg" alt="triangle" >
+      </section>
+      <section class="resources section resources_padding_top">
```

Comment:
Setting up `resources_padding_top` modifier seems a bit redundant in this case. Please consider removing it and moving styling to `resources`itself

Reply:
Done🙂